config/types/files.go

```diff
@@ -108,7 +111,21 @@ func init() {
 			}
 
 			if file.Contents.Local != "" {
-				contents, err := ioutil.ReadFile(file.Contents.Local)
+				// The provided local file path is relative to the value of the
+				// --files-dir flag.
+				filesDir := flag.Lookup("files-dir")
+				if filesDir == nil || filesDir.Value.String() == "" {
+					err := errors.New("Local files require setting the --files-dir flag to the directory that contains the file.")
```

Comment:
nit: other errors typically don't have sentence formatting. Can you lowercase `Local` and drop the period at the end?

Reply:
Fixed.